pkg/volume/fc/disk_manager.go

```diff
@@ -117,5 +119,44 @@ func diskTearDown(manager diskManager, c fcDiskUnmounter, volPath string, mounte
 		}
 	}
 	return nil
+}
+
+// utility to map a device to symbolic link
+func deviceSetUp(manager diskManager, b fcDiskMapper, volPath string) error {
+	globalPDPath := manager.MakeGlobalPDName(*b.fcDisk)
+	if !filepath.IsAbs(globalPDPath) || !filepath.IsAbs(volPath) {
+		return fmt.Errorf("These paths should be absolute: globalPDPath: %s, volPath: %s", globalPDPath, volPath)
+	}
+	devicePath, err := os.Readlink(globalPDPath + "/" + "symlink")
+	if err != nil {
+		glog.Errorf("failed to readlink: %s", globalPDPath+"/"+"symlink")
+		return err
+	}
+	if err != nil && !os.IsNotExist(err) {
```

Comment:
this check is always false, there is `if err != nil ` three lines above.

Reply:
This logic was removed